pkgs/tools/package-management/nixpkgs-review/default.nix

```diff
@@ -7,19 +7,21 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "nixpkgs-review";
-  version = "2.5.0";
+  version = "2.6.0";
 
   src = fetchFromGitHub {
     owner = "Mic92";
     repo = "nixpkgs-review";
     rev = version;
-    sha256 = "1k4i54j5if86qf9dmwm8ybfc4j7ap40y82f03hxfxb7lzq5cqmcv";
+    sha256 = "sha256-096oSvc9DidURGKE0FNEBOQz82+RGg6aJo8o9HhaSp0=";
   };
 
   makeWrapperArgs = [
     "--prefix" "PATH" ":" (lib.makeBinPath [ nixFlakes git ])
   ];
 
+  checkInputs = [ python3.pkgs.pytest nixFlakes git ];
```

Comment:
You cannot run tests inside nix sandbox actually anymore. This should just disable tests.